### DISCUSSION TITLE
Hacktribe Firmware  for the black Electribe Sample

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#162] [Q&A] [answered]
Hello

I will get from am friend an second Electribe. Super happy about this. I love the Electribes

What i wanted to know I modified the Hacktribe firmware for the Synth Electribe back then. I'll be getting an Electribe 2 sampler soon. Can the firmware simply be loaded onto the sampler or do I have to do it all over again? I already have a modified firmware on the desktop

Thank you so much for the help.

Greetings Max

**Comment by bangcorrupt** [comment, marked as answer]
For the sampler (red or black) don't modify the header when you hack the firmware.
